tests/unit/traits/has_general_helpers.php

```diff
@@ -85,18 +85,7 @@ public function get_course_config_params(array $overrideparams = []) {
     }
 
     public function update_system_config_value($configname, $newvalue) {
-        global $DB;
-
-        if ($record = $DB->get_record('config', ['name' => $configname])) {
-            $record->value = $newvalue;
-
-            $DB->update_record('config', $record);
-        } else {
-            $DB->insert_record('config', (object)[
-                'name' => $configname,
-                'value' => $newvalue,
-            ]);
-        }
+        set_config($configname, $newvalue);
```

Comment:
As this is a 1 line wrapper, can we extract this back and replace all the calls with a plain set_config instead?